README.md

```diff
@@ -124,10 +150,10 @@ npm-run-all package is added as a development dependency. It is used to run iOS
 
 ## WebDriverIo Capabilities 
 
-To run the same test based on different number of configurations and properties, add new elements under the capabilities array in
-**android.conf.js** or **iOS.conf.js**.
+To run the same test based on different number of configurations and properties, add new elements under the capabilities array in one of
+**android_native.conf.js**, **iOS_native.conf.js**, **android_web.conf.js** or **iOS_web.conf.js**.
```

Comment:
can you write something like "one of the conf files" instead of writing this line everywhere?